act/scio/api.py

```diff
@@ -48,6 +46,18 @@
 # pylint: disable=too-few-public-methods
 
 
```

Comment:
needed to use this instead of constr to get mypy happy:
https://stackoverflow.com/questions/66924001/conflict-between-pydantic-constr-and-mypy-checking